src/jvmTest/kotlin/com/github/michaelbull/result/RunTest.kt

```diff
@@ -0,0 +1,6 @@
+package com.github.michaelbull.result
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.runBlocking
+
+actual fun runTest(block: suspend (scope : CoroutineScope) -> Unit) = runBlocking { block(this) }
```

Comment:
We should probably rename this function to `runBlockingTest`, purely for consistency with that jvm-specific library naming.

Reply:
good call, done.